### DISCUSSION TITLE
Cycle lane overlay

### DISCUSSION THREAD

**Discussion by frankie2784** [streetcomplete/StreetComplete#4608] [Ideas]
First of all, thank you so much to the developers of this app. It's a fantastic way to contribute to OSM in real time!

In both a personal and professional capacity, I am primarily interested in mapping active transport-related attributes, especially cycling infrastructure. As someone working in the active transport data space in government, I can tell you that OSM is really one of the best sources of detailed cycling infrastructure around - so I am dedicated to helping improve this data in any way that I can.

To this end, are there any plans to develop a cycling overlay in SC, similar to the parking and sidewalk overlays, so that we can more easily add these attributes to specific ways we are interested in?

It would be particularly helpful if such an overlay showed where cycle lane infrastructure was already mapped in OSM to help target our efforts. To that end, similar indication of where street parking and footways are already tagged could make a great addition to those overlays.

I would be really grateful for thoughts and comments on this. And apologies if this has been discussed elsewhere.

Cheers,
Paul

**Comment by mnalis** [comment]
- see https://github.com/streetcomplete/StreetComplete/issues/4233

**Reply by frankie2784** [reply]
Glad to hear there is broad support for this 😊
